substrate/network-libp2p/src/network_state.rs

```diff
@@ -180,8 +181,21 @@ impl NetworkState {
 				PeersStorage::Json(peerstore)
 			} else {
 				warn!(target: "sub-libp2p", "Failed to open peer storage {:?} \
-					; peers won't be saved", path);
-				PeersStorage::Memory(MemoryPeerstore::empty())
+					; peers file will be reset", path);
+				fs::remove_file(&path).unwrap();
```

Comment:
codebase style is to use `expect` and prove that the unwrap will not trigger

Reply:
(in this case, I'm not sure that can be proven. you should try to handle the error somehow, maybe by logging it)

Reply:
ok I will adjust